haystack/nodes/prompt/invocation_layer/hugging_face.py

```diff
@@ -43,6 +43,15 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor, **kwa
             stop_result = torch.isin(self.stop_words["input_ids"], input_ids[-1])
             return any(all(stop_word) for stop_word in stop_result)
 
+    def get_task(model: str, use_auth_token: Optional[Union[str, bool]] = None, timeout: float = 3.0) -> Optional[str]:
+        """
+        Simplified version of transformers.pipelines.get_task with support for timeouts
+        """
+        try:
+            return model_info(model, token=use_auth_token, timeout=timeout).pipeline_tag
+        except Exception as e:
+            raise RuntimeError(f"The task of {model} could not be checked because of the following error: {e}")
```

Comment:
```suggestion
            raise RuntimeError(f"The task of {model} could not be checked because of the following error: {e}") from e
```

I feel it can be slightly improved. (Read [this](https://www.pythontutorial.net/python-oop/python-raise-from/))

Reply:
Thanks, I didn't know about this feature.